src/Sylius/Component/Core/OrderProcessing/OrderPaymentProcessor.php

```diff
@@ -62,7 +62,7 @@ public function process(BaseOrderInterface $order)
             return;
         }
 
-        $newPayment = $order->getLastNewPayment();
+        $newPayment = $order->getLastCartPayment();
```

Comment:
Yup 😄  and that's why if orders is ``completed``, payment has already ``create`` transition applied. This is of course a temporary solution - we should have two processors, for checkout and fail/cancel callbacks. Or should I implement them in this PR?